credsweeper/filters/value_dictionary_keyword_check.py

```diff
@@ -21,6 +22,6 @@ def run(self, line_data: LineData) -> bool:
         """
         if line_data.value is None:
             return True
-        if any(keyword in line_data.value.lower() for keyword in self.keyword_checklist.get_list()):
+        if any(keyword == line_data.value.lower() for keyword in self.keyword_checklist):
```

Comment:
Suppose, there should be equals, but many tests fail wit growing number of credentials.
@csh519, @meanrin what do you think?